test/connectors/Sub09ConnectorSpec.scala

```diff
@@ -133,6 +142,7 @@ class Sub09ConnectorSpec extends SpecBase {
     val companyInformation: CompanyInformation = CompanyInformation("Example Ltd", "1", AddressInformation("Example Rd", "Example", Some("AA00 0AA"), "GB"))
     val companyInformationNoConsentFalse: CompanyInformation = CompanyInformation("Example Ltd", "0", AddressInformation("Example Rd", "Example", Some("AA00 0AA"), "GB"))
     val xiEoriInformation: XiEoriInformation = XiEoriInformation("XI123456789000", "1", XiEoriAddressInformation("Example Rd", Some("Example"), Some("GB"), None, Some("AA00 0AA")))
+    val xiEoriInformationWithNoAddress: XiEoriInformation = XiEoriInformation("XI123456789000", "1", XiEoriAddressInformation(""))
```

Comment:
Could you please split the line after : XiEoriInformation = as it is very long line?

Reply:
Long lines are split to make it short and readable